spock/mcmap/mapdata.py

```diff
@@ -1,68 +1,71 @@
 from spock.utils import BoundingBox
 
-#Materials
-MCM_MAT_ROCK         = 0x00
-MCM_MAT_DIRT         = 0x01
-MCM_MAT_WOOD         = 0x02
-MCM_MAT_WEB          = 0x03
-MCM_MAT_WOOL         = 0x04
-MCM_MAT_VINE         = 0x05
-MCM_MAT_LEAVES       = 0x06
-
-#Gate
-MCM_GATE_SOUTH       = 0x00
-MCM_GATE_WEST        = 0x01
-MCM_GATE_NORTH       = 0x02
-MCM_GATE_EAST        = 0x03
-
-MCM_GATE_CLOSE       = 0x00
-MCM_GATE_OPEN        = 0x01
-
-MCM_GATE_UNPOWERED   = 0x00
-MCM_GATE_POWERED     = 0x01
-
-#Door
-MCM_DOOR_WEST        = 0x00
-MCM_DOOR_NORTH       = 0x01
-MCM_DOOR_EAST        = 0x02
-MCM_DOOR_SOUTH       = 0x03
-
-MCM_DOOR_CLOSE       = 0x00
-MCM_DOOR_OPEN        = 0x01
-
-MCM_DOOR_LOWER       = 0x00
-MCM_DOOR_UPPER       = 0x01
-
-MCM_DOOR_HINGE_LEFT  = 0x00
+# Materials
+MCM_MAT_ROCK = 0x00
+MCM_MAT_DIRT = 0x01
+MCM_MAT_WOOD = 0x02
+MCM_MAT_WEB = 0x03
+MCM_MAT_WOOL = 0x04
+MCM_MAT_VINE = 0x05
+MCM_MAT_LEAVES = 0x06
+
+# Gate
+MCM_GATE_SOUTH = 0x00
+MCM_GATE_WEST = 0x01
+MCM_GATE_NORTH = 0x02
+MCM_GATE_EAST = 0x03
+
+MCM_GATE_CLOSE = 0x00
+MCM_GATE_OPEN = 0x01
+
+MCM_GATE_UNPOWERED = 0x00
+MCM_GATE_POWERED = 0x01
+
+# Door
+MCM_DOOR_WEST = 0x00
+MCM_DOOR_NORTH = 0x01
+MCM_DOOR_EAST = 0x02
+MCM_DOOR_SOUTH = 0x03
+
+MCM_DOOR_CLOSE = 0x00
+MCM_DOOR_OPEN = 0x01
+
+MCM_DOOR_LOWER = 0x00
+MCM_DOOR_UPPER = 0x01
+
```

Comment:
haha yeah, well, what you usually do with this is to generate namedtuple() so that you can actually use those values from a tuple, without having loose vars.